xml/dm_registers.xml

```diff
@@ -143,11 +143,11 @@
         <field name="0" bits="15:2" access="R" reset="0" />
         <field name="ndmreset" bits="1" access="R/W" reset="0">
             This bit controls the reset signal from the DM to the rest of the
-            system. To perform a system reset the debugger writes 1,
+            system. The signal should reset every part of the system, including
+            every hart, except for the DM.
```

Comment:
what about the debug transport? should we just say "the debug logic, including the Debug Module and Debug Transport Module"

Reply:
I'm ambivalent about that. The reset could or could not reset the transport. We should be clear about it, though. Sounds like you would prefer for the transport not to be reset. But what if the transport is something used by the reset of the system as well (like some shared PCI bus)? I'm inclined to add something like:
"This may include the debug transport. The debugger must be prepared to reestablish the connection to the Debug Module immediately after writing 1 to ndmreset."

Reply:
Ok. Well I guess it gets back to the "how do you then clear ndmreset" if the transport is reset.

Reply:
Right. I forgot about that. I'll add something explicit to the spec about that.

Reply:
Or is this an argument to make the register handle the entire reset sequence and clear itself?